pyproject.toml

```diff
@@ -2,6 +2,84 @@
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[project]
+name = "pyramid"
+version = "2.1.dev0"
+description = "The Pyramid Web Framework, a Pylons project"
+
+authors = [
+  {name = "Chris McDonough, Agendaless Consulting", email = "pylons-discuss@googlegroups.com"},
+]
+
+# license = "BSD-derived (Repoze)"
+license-files = ["LICENSE.txt"]
+
+readme = {file = "README.rst", content-type = "text/x-rst"}
+
+requires-python = ">=3.9"
+dependencies = [
+    "hupper >= 1.5",  # ignore_files support
+    "plaster",
+    "plaster_pastedeploy",
+    "setuptools",
+    "translationstring >= 0.4",  # py3 compat
+    "venusian >= 1.0",  # ``ignore``
+    "webob >= 1.8.3",  # Accept.parse_offer
+    "zope.deprecation >= 3.5.0",  # py3 compat
+    "zope.interface >= 3.8.0",  # has zope.interface.registry
+]
+
+keywords = ["web", "wsgi", "pylons", "pyramid"]
+classifiers = [
+        "Development Status :: 6 - Mature",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Framework :: Pyramid",
+        "Topic :: Internet :: WWW/HTTP",
+        "Topic :: Internet :: WWW/HTTP :: WSGI",
+        "License :: Repoze Public License",
+]
+
+[tool.setuptools.package-data]
+mypkg = ["*.txt", "*.rst"]
+
+[project.scripts]
+pserve = "pyramid.scripts.pserve:main"
+pshell = "pyramid.scripts.pshell:main"
+proutes = "pyramid.scripts.proutes:main"
+pviews = "pyramid.scripts.pviews:main"
+ptweens = "pyramid.scripts.ptweens:main"
+prequest = "pyramid.scripts.prequest:main"
+pdistreport = "pyramid.scripts.pdistreport:main"
+
+[project.entry-points."paste.server_runner"]
+tomatoes = "spam:main_tomatoes"
```

Comment:
example left in?

Reply:
oops. thanks.

I'm a bit surprised that that didn't give me an error when built, but oh well.

Reply:
Makes sense to me: nothing happens at build time except the entry points being recorded for run-time use.

It would be different for scripts entry-points.